src/main/java/net/testudobank/MvcController.java

```diff
@@ -381,64 +381,28 @@ public String submitDispute(@ModelAttribute("user") User user) {
     int userOverdraftBalanceInPennies = TestudoBankRepository.getCustomerOverdraftBalanceInPennies(jdbcTemplate, userID);
 
     int reversalAmountInPennies = (int) logToReverse.get("Amount");
+    int reversalAmount = reversalAmountInPennies / 100;
 
     // If transaction to reverse is a deposit, then withdraw the money out
     if (((String) logToReverse.get("Action")).toLowerCase().equals("deposit")) {
       // if withdraw would exceed max overdraft possible, return welcome
       if (userOverdraftBalanceInPennies + (reversalAmountInPennies - userBalanceInPennies) > MAX_OVERDRAFT_IN_PENNIES) {
         return "welcome";
       }
+      user.setAmountToWithdraw(reversalAmount);
+      submitWithdraw(user);
 
-      // if balance is large enough to have reversalAmount taken from it, subtract reversalAmount from balance
-      if (userBalanceInPennies - reversalAmountInPennies > 0){
-        TestudoBankRepository.decreaseCustomerBalance(jdbcTemplate, userID, reversalAmountInPennies);
-      } else { // Case when reversing deposit causes overdraft or go deeper into overdraft
-        // Set main balance to 0 since we are either going into overdraft or already in overdraft
-        TestudoBankRepository.setCustomerBalance(jdbcTemplate, userID, 0);
-
-        int differenceInPennies = reversalAmountInPennies - userBalanceInPennies;
-
+      if (userBalanceInPennies - reversalAmountInPennies <= 0){
         //check if deposit helped pay off overdraft balance
```

Comment:
Edit this comment to say `//check if the reversed deposit helped pay off overdraft balance`

Reply:
Sure, I'll change the comment to better reflect the condition.